gazebo_ros/launch/gzserver.launch.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Launch a Gazebo server with command line arguments."""
+
+from os import environ
+
+from scripts import GazeboRosPaths
+
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.actions import ExecuteProcess
+from launch.substitutions import LaunchConfiguration
+from launch.substitutions import PythonExpression
+
+
+def generate_launch_description():
+    model, plugin, media = GazeboRosPaths.get_paths()
+
+    if 'GAZEBO_MODEL_PATH' in environ:
+        model += ':'+environ['GAZEBO_MODEL_PATH']
+    if 'GAZEBO_PLUGIN_PATH' in environ:
+        plugin += ':'+environ['GAZEBO_PLUGIN_PATH']
+    if 'GAZEBO_RESOURCE_PATH' in environ:
+        media += ':'+environ['GAZEBO_RESOURCE_PATH']
+
+    env = {'GAZEBO_MODEL_PATH': model,
+           'GAZEBO_PLUGIN_PATH': plugin,
+           'GAZEBO_RESOURCE_PATH': media}
+
+    return LaunchDescription([
+        DeclareLaunchArgument('world', default_value='worlds/empty.world',
```

Comment:
How about leaving the default empty so Gazebo decides on the default value?

Reply:
The problem with leaving it empty is that after an empty string, `gazebo` tries to load a world named '' and it falls back to default but shows a warning world name '' not found.